clang-tools-extra/clang-tidy/portability/AvoidPlatformSpecificFundamentalTypesCheck.cpp

```diff
@@ -0,0 +1,183 @@
+//===--- AvoidPlatformSpecificFundamentalTypesCheck.cpp - clang-tidy ------===//
```

Comment:
```suggestion
//===----------------------------------------------------------------------===//
```

Emacs tags a no longer needed. See #153942.